src/avatar.jsx

```diff
@@ -197,15 +197,27 @@ class Avatar extends React.Component {
     const originalWidth = this.image.width;
     const originalHeight = this.image.height;
     const ration = originalHeight / originalWidth;
-    const height = this.props.height || originalHeight;
+    let imageHeight;
+    let imageWidth;
+    if (this.props.imageHeight && !this.props.imageWidth) {
+      imageHeight = this.props.imageHeight || originalHeight;
+      imageWidth = imageHeight / ration;
+    } else if (this.props.imageWidth && !this.props.imageHeight) {
+      imageWidth = this.props.imageWidth;
+      imageHeight = imageWidth * ration || originalHeight;
+    } else if (this.props.imageHeight && this.props.imageWidth) {
+      throw new Error('The imageWidth and imageHeight properties can not be set together.');
```

Comment:
@FlorianSW I think that it would be better to setup default behaviour instead of throwing new error. For example use imageWidth. What do you think ?

Reply:
Sorry for my late reply :(

Sounds good for me, too, however, I would issue a console warning, that passing both values is not supported, and that imageWidth will be used, only, ok? :)